File: Android/TRTCScenesDemo/trtcmeetingdemo/src/main/java/com/tencent/liteav/meeting/model/TRTCMeetingDef.java
package com.tencent.liteav.meeting.model;

public class TRTCMeetingDef {
    public static class UserInfo {
        public String  userId;
        public String  userName;
        public String  userAvatar;
        // 用户是否打开了视频
        public boolean isVideoAvailable;
        // 用户是否打开音频
        public boolean isAudioAvailable;
        // 是否对用户静画
        public boolean isMuteVideo;
        // 是否对用户静音
        public boolean isMuteAudio;

        public UserInfo() {
            userId = "";
            isVideoAvailable = false;
            isAudioAvailable = false;
            isMuteVideo = false;
            isMuteAudio = false;
        }
    }
}

File: iOS/TRTCScenesDemo/TXLiteAVDemo/TRTCMeetingDemo/model/impl/TRTCMeetingDef.m
//
//  TRTCMeetingDef.m
//  TRTCScenesDemo
//
//  Created by xcoderliu on 4/21/20.
//  Copyright © 2020 xcoderliu. All rights reserved.
//
#import <Foundation/Foundation.h>
#import "TRTCMeetingDef.h"

@implementation TXUserInfo

- (instancetype)init {
    if (self = [super init]) {
        self.userId = @"";
        self.userName = @"";
        self.avatarURL = @"";
    }
    return self;
}

@end


@implementation TRTCMeetingUserInfo

- (instancetype)init {
    if (self = [super init]) {
        self.isVideoAvailable = NO;
        self.isAudioAvailable = NO;
        self.isMuteAudio = NO;
        self.isMuteVideo = NO;
    }
    return self;
}

@end
